### DISCUSSION TITLE
read but no write

### DISCUSSION THREAD

**Discussion by 82niko** [alandtse/tesla#841] [Q&A]
Hi all
I am not that skilled in HA, sorry :-)
The problem is that the car will not react to "write" signals, like horn or light. But HA can read info because I get map position and all the information from the car. It has been working but all off the sudden it stopped.
Token is refreshed, all is updatet, I checked Tesla home page, restart, tried all.

**Comment by skipishere** [comment]
This sounds like a duplicate of https://github.com/alandtse/tesla/issues/743 & discussed at length in https://github.com/alandtse/tesla/issues/774

**Comment by 82niko** [comment]
hmm. . so no easy work around

**Reply by skipishere** [reply]
At the moment, sadly not. Very much a 'watch this space'
